README.md

```diff
@@ -56,6 +73,13 @@ scrolling the area by dragging its empty space, but keep the
 opportunity to select the text (see
 [example](http://asvd.github.io/jailed/demos/web/process/)).
 
--
+Thank you for use this lib.
+
+TO DO
```

Comment:
I'm not sure if you need TODO's on the readme (though I've seen it many times in other projects)